.github/workflows/ci_test.yml

```diff
@@ -6,6 +6,7 @@ on:
     branches:
       - master
       - develop
+      - logging-improvement  # TODO Remove this after testing is complete
```

Comment:
@mwartell, we need to remove this for the final merge

Reply:
done.